src/main/java/com/booster/cliclient/command/Command.java

```diff
@@ -26,4 +27,9 @@ public static Command from(String in) {
                 .orElse(UNDEFINED);
     }
 
+    @Override
+    public String toString() {
+        return this.name() + "(" + getValue() + ")";
```

Comment:
With such implementation, you have the following output
``` 
>> EXIT(e)
```
I think the following variant will be more user-friendly:
```
e - exit
```